tasks.py

```diff
@@ -190,103 +211,66 @@ def docker_l10n_update(ctx):
 @task
 def docker_test_python(ctx):
     """Run python tests"""
-    print("Running flake8")
+    print("* Running flake8")
     ctx.run("docker-compose run --rm backend pipenv run flake8 tasks.py network-api", **PLATFORM_ARG)
-    print("Running tests")
+    print("* Running tests")
     docker_manage(ctx, "test")
 
 
 @task
 def docker_test_node(ctx):
     """Run node tests"""
-    print("Running tests")
+    print("* Running tests")
     ctx.run("docker-compose run --rm watch-static-files npm run test", **PLATFORM_ARG)
 
 
-@task
-def docker_switching_branch(ctx):
-    """Get a new database with fake data and rebuild images"""
-    print("Stopping services first")
-    ctx.run("docker-compose down")
-    print("Deleting database")
-    ctx.run("docker volume rm foundationmozillaorg_postgres_data")
-    print("Rebuilding images and install dependencies")
-    ctx.run("docker-compose build")
-    print("Applying database migrations.")
-    docker_migrate(ctx)
-    print("Creating fake data")
-    docker_manage(ctx, "load_fake_data")
-    print("Updating localizable fields")
-    docker_l10n_sync(ctx)
-    docker_l10n_update(ctx)
-    print("Updating block information")
-    docker_manage(ctx, "block_inventory")
-
-
 @task
 def docker_new_db(ctx):
     """Delete your database and create a new one with fake data"""
-    print("Stopping services first")
+    print("* Stopping services first")
     ctx.run("docker-compose down")
-    print("Deleting database")
+    print("* Deleting database")
     ctx.run("docker volume rm foundationmozillaorg_postgres_data")
-    print("Applying database migrations.")
+    print("* Applying database migrations.")
     docker_migrate(ctx)
-    print("Creating fake data")
+    print("* Creating fake data")
     docker_manage(ctx, "load_fake_data")
-    print("Updating localizable fields")
-    docker_l10n_sync(ctx)
-    docker_l10n_update(ctx)
-    print("Updating block information")
-    docker_manage(ctx, "block_inventory")
+    docker_l10n_block_inventory(ctx)
+    docker_create_super_user(ctx)
 
 
 @task(aliases=["docker-catchup"])
 def docker_catch_up(ctx):
     """Rebuild images and apply migrations"""
-    print("Stopping services first")
+    print("* Stopping services first")
     ctx.run("docker-compose down")
-    print("Rebuilding images and install dependencies")
+    print("* Rebuilding images and install dependencies")
     ctx.run("docker-compose build")
-    print("Applying database migrations.")
+    print("* Applying database migrations.")
     docker_migrate(ctx)
-    print("Updating localizable fields")
-    docker_l10n_sync(ctx)
-    docker_l10n_update(ctx)
-    print("Updating block information")
-    docker_manage(ctx, "block_inventory")
+    docker_l10n_block_inventory(ctx)
 
 
 @task
-def docker_setup(ctx):
-    """Prepare your dev environment after a fresh git clone"""
+def docker_new_env(ctx):
```

Comment:
This is probably a crazy question, but: does docker have the ability to set some default named container? Or, is it easy to make a run task change the docker file so that it changes the name/id of the docker container to start?

Specifically, I was wondering whether it would it be possible to have an `inv docker-switch-env [name]` that either loads a named env, or falls through to the `new-env` task in order to build a new environment with its own database if it doesn't exist yet, so that if people are working on several issues at the same time with different migration chains, they can switch environments in a way that keeps the code and db in sync? 

(the previous `switch` task would build a new db by clearing the existing db - I'm wondering if there's a way to get "build new" if there's nothing, but "reuse" if there was something, with `new-env` always clearing if it exists, before building a new env, but `switch-env` only changing the dockerfile so that `docker-compose up` will load the correct container, rather than build anything, if that container exists?)

Reply:
That's not something that I want to investigate. Maybe we could do something using image tags but I don't have the time to look into this.